src/Rule/UrlHandler.php

```diff
@@ -27,9 +27,17 @@ public function validate(mixed $value, object $rule, ValidationContext $context)
         }
 
         $result = new Result();
+        if (!is_string($value)) {
+            $result->addError($rule->getMessage(), [
+                'attribute' => $context->getAttribute(),
+                'valueType' => get_debug_type($value),
```

Comment:
For same message must be fixed attributes list.

Reply:
https://github.com/yiisoft/validator/pull/364#discussion_r1013909544